src/AWSCore.jl

```diff
@@ -203,7 +203,10 @@ Service endpoint URL for `request`.
 function service_url(aws, request)
     endpoint = get(request, :endpoint, request[:service])
     region = "." * aws[:region]
-    if endpoint == "iam" || (endpoint == "sdb" && region == ".us-east-1") || region=="."
+
+    regionless_service_endpoints = ["iam", "route53"]
```

Comment:
Slightly shorter name and we may as well use a `tuple` as this doesn't need to be mutated:
```suggestion
    regionless_endpoints = ("iam", "route53")
```